docs/pages/versions/unversioned/sdk/screen-capture.md

```diff
@@ -88,6 +88,14 @@ import {
 
 ## Methods
 
+### `isAvailableAsync()`
+
+Returns whether the Screen Capture API is enabled on the current device.
```

Comment:
```suggestion
Returns whether the Screen Capture API is available on the current device.
```

Reply:
@EvanBacon, how do you feel about replacing `enabled` with `available`? As @bbarthec mentioned, this might give a sense that it could be enabled or disabled manually.

Reply:
I think is available should refer to if the capability is available to use in the app. If there is an enabled/disabled state, that should be in a different method.